.env

```diff
@@ -1,16 +1,22 @@
-# influx2 variables
-DOCKER_INFLUXDB_INIT_MODE=setup
-DOCKER_INFLUXDB_INIT_USERNAME=k6
-DOCKER_INFLUXDB_INIT_PASSWORD=telegraf
-DOCKER_INFLUXDB_INIT_ORG=my-org
-DOCKER_INFLUXDB_INIT_BUCKET=my-bucket
-DOCKER_INFLUXDB_INIT_ADMIN_TOKEN=specialToken000
+# THis file is used in docker-compose.yml and sets the env variables.
 
-# necessary to use the influx2 export of xk6
-K6_INFLUXDB_ORGANIZATION=my-org
-K6_INFLUXDB_BUCKET=my-bucket
-K6_INFLUXDB_TOKEN=specialToken000
+# PORTS <HOST:CONTAINER>
+DQANALYZER_PORT="9090"
+DQAPI_PORT="8099"
+DQTRANSLATOR_PORT="8080"
+DQEXEC_PORT="8090"
+MONGO_PORT="27017"
+MONGO_MANAGEMENT_PORT="8081"
+INFLUX_PORT="8086"
+GRAFANA_PORT="3030:3000"
```

Comment:
Getting runtime errors when executing the docker compose file. Probably here:
* error decoding 'ports': Invalid ip address: 3030